scripts/common/rook.sh

```diff
@@ -225,3 +225,17 @@ function current_ceph_version() {
     kubectl -n rook-ceph get deployment rook-ceph-mgr-a -o jsonpath='{.metadata.labels.ceph-version}' 2>/dev/null \
         | awk -F'-' '{ print $1 }'
 }
+
+function rook_cephcluster_ready() {
+    local ceph_cluster_phase=
+    local ceph_cluster_status=
+    ceph_cluster_phase=$(kubectl -n rook-ceph get cephcluster rook-ceph --template '{{.status.phase}}')
+    if [  "$ceph_cluster_phase" = "Ready"  ]; then
+        ceph_cluster_status=$(kubectl -n rook-ceph get cephcluster rook-ceph --template '{{.status.ceph.health}}')
+        if [ "$CEPH_POOL_REPLICAS" -gt 1 ] && [ "$ceph_cluster_status" = "HEALTH_WARN" ]; then
```

Comment:
We cannot to do that because we have many warns that we need to ignore.
For example, if we install kurl in a host that has only 1 master node we will see warnings about the pools. 

See we ignoring some warnings: https://github.com/replicatedhq/kURL/blob/main/pkg/rook/health.go#L66-L125

Also, see that we check the ceph status already: https://github.com/replicatedhq/kURL/blob/2f119ae38bc88a5ab4fc8ad1a6d14aa7cb674fbe/addons/rook/1.9.12/install.sh#L330-L335

Reply:
https://github.com/replicatedhq/kURL/blob/2f119ae38bc88a5ab4fc8ad1a6d14aa7cb674fbe/addons/rook/1.9.12/install.sh#L307-L324

Reply:
As an alternative you can use Kurl rook wait-for-health command

Reply:
> As an alternative you can use Kurl rook wait-for-health command

But we do those checks already. So, how could that sort out the problem? 
Rook creates a lot of pods, so are not those others that might be with ContainerCreating, etc. status? 
So, I think we just can ensure that all pods in the namespace are with status as completed and running.

Reply:
In my initial comment I had hoped that the cephcluster status.phase or conditions would be enough to check for rook ready, but it seems that the pod check is still required. if so you can forgo my request.

Reply:
Closing this one I think it is addressed / discussed now.